packages/api/src/parse-cli.ts

```diff
@@ -316,6 +316,11 @@ export default function parseCli(argv?: string | readonly string[]) {
           "base stream name to be used in wildcard-based routing scheme.",
         type: "string",
       },
+      "recording-source-url": {
+        describe:
+          "base stream name to be used in wildcard-based routing scheme.",
+        type: "string",
+      },
```

Comment:
WDYT of doing this similarly to other VOD object stores? Meaning we create a specific object store in the API and then this field is only a `recordingSourceObjectStoreId`, with the actual url/publicUrl being fetched from the DB whenever necessary. We will only be using the public URL in the API for now, but I think we may evolve this in different directions (e.g. catalyst asking the "write" URL from Studio) and the consistency is good IMO.

I can help creating the object store object if you wanna follow this approach! Gimme the URL/publicURL pair and I can give you the ID.

Reply:
Updated in be2a1f64d7ccf309584a69035295d88989833208

Reply:
Not yet, but I'll create them (or ask for your help) before merging this PR.